doc/man3/OSSL_PROVIDER.pod

```diff
@@ -92,7 +92,11 @@ for use.
 
 OSSL_PROVIDER_do_all() iterates over all loaded providers, calling
 I<cb> for each one, with the current provider in I<provider> and the
-I<cbdata> that comes from the caller.
+I<cbdata> that comes from the caller. If no other provider has been loaded
+before calling this function, the default provider is still available as
+fall-back.
```

Comment:
I think we've tended to use the word "fallback", i.e. without a hyphen

Reply:
Well, only in man7: There it's "fallback" throughout; in man3 it's been mostly "fall-back". But I also like "fallback" better. OK to change the other "fall-back" instances too?

Reply:
If you change fall-back to fallback, consider a PR that adds this to the `preferred_words` list in `util/find-doc-nits`

Reply:
Didn't know this existed: Thanks for pointing to it. So done.